content/en/blog/2020/alb-ingress-gateway-iks/index.md

```diff
@@ -239,9 +222,7 @@ You use the certificates and the keys provided to you for the ingress gateway an
           protocol: HTTPS
         tls:
           mode: MUTUAL
-          serverCertificate: /etc/istio/ingressgateway-certs/tls.crt
-          privateKey: /etc/istio/ingressgateway-certs/tls.key
-          caCertificates: /etc/istio/ingressgateway-ca-certs/trustid-x3-root.pem
+          credentialName: $INGRESS_GATEWAY_SECRET
```

Comment:
so ca certificates are not required here? Does it work without them?

Reply:
You don't need to specify the location of the ca certs in the Gateway yaml  if they are located in `$INGRESS_GATEWAY_SECRET-cacert`. See the last line in this section: https://istio.io/latest/docs/tasks/traffic-management/ingress/secure-ingress/#configure-a-mutual-tls-ingress-gateway

Reply:
I see, did not know that.